App.js

```diff
@@ -1,19 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import LogoSvg from "./LogoSvg";
-import Quotes from "./Quotes";
 import AnimateSvg from './AnimateSvg';
 import AnimateQuotes from './AnimateQuotes';
+import { useIsShake } from './useIsShake';
 
 export default function App() {
+  const [hasToggledBefore, setHasToggledBefore] = useState(false);
```

Comment:
State for indicating if a shake has ever registered so that animation components will run appropriate animation compositions.